examples/fluids/src/setupdm.c

```diff
@@ -24,12 +24,14 @@ PetscErrorCode CreateDistributedDM(MPI_Comm comm, ProblemData *problem,
                                    SetupContext setup_ctx, DM *dm) {
   DM               dist_mesh = NULL;
   PetscPartitioner part;
-  PetscInt         dim = problem->dim;
+  PetscInt         dim = problem->dim, faces[3] = {3, 3, 3};
```

Comment:
Does this mean three faces per each dimension? I am not sure if this understanding is correct since I think the default was two faces per dimension?!

Reply:
Hmm, I'm not seeing where that would be specified. When Matt refactored DMPlex creation, `DMPlexCreateBoxMesh` no longer calls `DMSetFromOptions` and thus doesn't process options like `-dm_plex_box_faces`. Instead, you can now do everything (read in or create any kind of mesh, distribute, interpolate, refine, extrude) from three lines
```c
DMCreate(comm, &dm);
DMSetType(dm, DMPLEX);
DMSetFromOptions(dm);
```
This will simplify our code, but I figured we'd wait for 3.16 to be released before shifting over to this pattern.

Reply:
I'd just copied the 3,3,3 default from solids, but it was probably 1,1,1 in fluids (not that that's likely to be very helpful).

Reply:
The `faces` array is asking how many faces you want on each side of the box. So `[2, 3, 5]` would give 30 hex cells in the box.

Reply:
Oh, then I was confusing two separate things! I thought that was determining number of box faces which was 6 for 3D. I completely forgot for a moment that the mesh is a _box_ and the number of box faces is fixed! :laughing:

Reply:
One reason to have 3,3,3 as a default is that even with full Dirichlet boundary conditions and basis degree 1, _something_ can happen.

Reply:
Yes, `3,3,3` makes more sense @jedbrown. 

Thanks, @jedbrown and @knepley for the clarification!